Packs/LogRhythmRest/Integrations/LogRhythmRest/LogRhythmRest.py

```diff
@@ -583,6 +584,1316 @@ def get_alarm_events(data_args):
+        "Syslog - ZXT Load Balancer":1000411,
+        "Syslog - ZyWALL VPN Firewall":1000666,
+        "Syslog Avaya G450 Media Gateway":1000670,
+        "Syslog File - AIX Host":1000006,
+        "Syslog File - BSD Format":35,
+        "Syslog File - HP-UX Host":1000145,
+        "Syslog File - IRIX Host":1000295,
+        "Syslog File - Linux Host":103,
+        "Syslog File - LogRhythm Syslog Generator":13,
+        "Syslog File - MS 2003 Event Log (Snare)":1000039,
+        "Syslog File - Oracle 10g Audit Trail":1000072,
+        "Syslog File - Oracle 11g Audit Trail":1000222,
+        "Syslog File - Solaris Host":104,
+        "UDLA - CA Single Sign-On":1000636,
+        "UDLA - Deepnet DualShield":1000286,
+        "UDLA - Drupal":1000496,
+        "UDLA - Finacle Core":1000196,
+        "UDLA - Finacle Treasury Logs":1000178,
+        "UDLA - Forcepoint":1000020,
+        "UDLA - Gallagher Command Centre":1000810,
+        "UDLA - iManage Worksite":1000732,
+        "UDLA - ISS Proventia SiteProtector - IPS":1000034,
+        "UDLA - LogRhythm Enterprise Monitoring Solution":1000314,
+        "UDLA - LREnhancedAudit":1000548,
+        "UDLA - McAfee ePolicy Orchestrator - Universal ePOEvents":1000788,
+        "UDLA - McAfee ePolicy Orchestrator 3.6 - Events":158,
+        "UDLA - McAfee ePolicy Orchestrator 4.0 - ePOEvents":1000079,
+        "UDLA - McAfee ePolicy Orchestrator 4.5 - ePOEvents":1000175,
+        "UDLA - McAfee ePolicy Orchestrator 5.0 - ePOEvents":1000531,
+        "UDLA - McAfee ePolicy Orchestrator 5.1 - ePOEvents":1000550,
+        "UDLA - McAfee ePolicy Orchestrator 5.3 - ePOEvents":1000696,
+        "UDLA - McAfee ePolicy Orchestrator 5.9 - ePOEvents":1000761,
+        "UDLA - McAfee Network Access Control":1000055,
+        "UDLA - McAfee Network Security Manager":1000453,
+        "UDLA - Microsoft System Center 2012 Endpoint Protection":1000587,
+        "UDLA - ObserveIT":1000605,
+        "UDLA - Oracle 10g Audit Trail":152,
+        "UDLA - Oracle 11g Audit Trail":1000171,
+        "UDLA - Oracle 12C Unified Auditing":1000658,
+        "UDLA - Oracle 9i Audit Trail":1000040,
+        "UDLA - Other":1000576,
+        "UDLA - SEL 3530 RTAC":1000285,
+        "UDLA - SharePoint 2007 AuditData":1000208,
+        "UDLA - SharePoint 2010 EventData":1000415,
+        "UDLA - SharePoint 2013 EventData":1000606,
+        "UDLA - Siemens Invision":1000229,
+        "UDLA - Sophos Anti-Virus":1000090,
+        "UDLA - Sophos Endpoint Security and Control":1000735,
+        "UDLA - Symantec CSP":1000505,
+        "UDLA - Symantec SEP":1000520,
+        "UDLA - Symmetry Access Control":1000270,
+        "UDLA - VMWare vCenter Server":1000378,
+        "UDLA - VMWare vCloud":1000538,
+        "VLS - Syslog - Infoblox - DNS RPZ":1000643,
+        "VLS - Syslog - Infoblox - Threat Protection":1000642
+        }
+    #Create filter query
+    query = []
+
+    if host_name != None:
```

Comment:
```suggestion
    if host_name:
```

do the same for the others